guides/source/active_record_callbacks.md

```diff
@@ -695,7 +1033,7 @@ User was saved to database
 
 By default, callbacks will run in the order they are defined. However, when
 defining multiple transactional `after_` callbacks (`after_commit`,
-`after_rollback`, etc), the order could be reversed from when they are defined.
+`after_rollback`, etc), the order can be reversed from when they are defined.
```

Comment:
This got me wondering if we should document this the other way around now, the default going forward is always in the order they're defined, but previous versions might reverse, and the configuration allows that behavior to be changed.

Reply:
@carlosantoniodasilva apologies I'm not sure what you mean here, can you expand?

Reply:
This makes sense, I took a stab at it!